custom/icds_reports/data_pull/queries.py

```diff
@@ -10,9 +10,10 @@
 class BaseQuery:
     query_file_path = ""
     setup_sql_file_path = ""
+    setup_table = ""
 
     def run(self, db_alias):
-        if self.setup_sql_file_path:
+        if self.setup_sql_file_path and self.setup_table:
```

Comment:
is this additional check needed?
It does make sense to check for that the table name is present if there is a file path

If you need that then a better way would be to raise an exception/error during instantiation?

Reply:
yes, it's better to keep that because I need that during the creation of that temp table and the same name is required during the dropping

Reply:
yup, i understand it's needed. I am just suggesting to satisfy that requirement differently.

So currently if someone makes a class with the `setup_sql_file_path` without the table name, and then run the data pull, it would just skip it and the user would have no clue why the data was not pulled. Raising an exception/error would make it explicit.